frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -92,5 +95,17 @@ class OverrideForkPointDialog(
         MutableProperty(::myOverrideOption) { myOverrideOption = it },
         OverrideOption::class.java
       )
+
+    row("The fork point commit:") {
+      comboBox(
```

Comment:
Hmmm for the sake of consistency... would it be possible to rather have a single `radioButton` for each commit? with parent comment and inferred commit highlighted as suggested options

Reply:
Do we still need the existing descriptions in front of the radioButtons?

Reply:
🤔  could you attach the screenshot of how the entire dialog looks for the current PR state?

Reply:
I'm trying to get it compile, first. It seems that some null value in the combobox is getting passed to the  overrided method of the custom Renderer, which leads to a null pointer exception. I am trying to debug that first.

Reply:
currently, at runtime, some null value is getting passed from the combobox to the overrided method of the cell renderer, leading to a null pointer exception.

Reply:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/109077949/192721186-44e00019-535d-4521-a25d-427300588309.png">

Reply:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/109077949/192728444-52fcc83f-5072-472c-aec7-c2d6f29ac953.png">
latest

Reply:
Okay... @mkondratek as you're a local UXer, could you suggest where could go from there? I've got some ideas but don't want to suggest anything yet

Reply:
Hey @mkondratek 👋

Reply:
whoops, Idk why but I did not receive email notifications from this one. maybe I must be added as a reviewer? dunno. feel free to ping me directly sooner if I am not responding 😆

Reply:
Hmmm, I think that the parent branch and the inferred commit are usually the right ones and we should not hide them among the other commits. I would rather keep them.

My idea is to have 3 radio buttons:
 [ ] parent branch 
 [ ] inferred commit
 [ ] manually pointed commit <combo box with all commits>


wdyt?

Reply:
☝🏻 It was indeed the original Ina's idea... although it also looked confusing to me (as inferred commit is then displayed twice: first highlighted, and then listed among other non-highlighted commits).

I'd rather see **a single list**, but:
1. scrollable
2. with some rich(er) formatting (not just combobox with plain unformatted text)
3. with parent branch and inferred commit clearly highlighted

WDYT?

Reply:
Sometimes also the selected commit in the combobox is invalid as a fork point.

Reply:
In the latest commit of this branch, the inferred commit and the parent one are at the top of the list of the combobox. and labeled as parent and inferred. WDYT? what can we do about the richer format or can we put them separately?

Reply:
@PawelLipski I am not convinced to this idea. I would rather list some commits twice than abandon the clear indication of what are the most possible commits to pick

and I cannot imagine that list containing all that information within. I mean - ofc we can bold smth, add `[tag-telling-what-I-am]` etc, but I doubt it will remain looking clearly and nice

Reply:
> Sometimes also the selected commit in the combobox is invalid as a fork point. We should be filtering out such commits, shouldn't we?

Hmmm what specifically do you mean by _invalid as fork point_? 🤔

Reply:
> I am not convinced to this idea. I would rather list some commits twice than abandon the clear indication of what are the most possible commits to pick

Okay, you've got me convinced/outvoted 😅  Let's go 3 radio buttons then, and some list (not necessarily combobox, better sth that allows for rich formatting) for the _manually pointed commit_ option

Reply:
> not necessarily combobox, better sth that allows for rich formatting

@mkondratek any recommendations?

Reply:
@PawelLipski sorry, I wanted to reply to your comment, and by mistake I ended up editing it somehow 😬

Reply:
> Hmmm what specifically do you mean by invalid as fork point? 🤔

I would repro the error in the plugin and let you know

Reply:
> > not necessarily combobox, better sth that allows for rich formatting
> 
> @mkondratek any recommendations?

@vzmerr idk, while looking for components/ui ideas I often depend on what I remember from the IDE, find it out in the intellij code and apply similar things in our code. you may want to take a look into https://plugins.jetbrains.com/docs/intellij/kotlin-ui-dsl-version-2.html too. for me combobox sounds most reasonable.

iiuc @PawelLipski would be happier with a list (seems to be anti-patternish to me, but anyway). this idea basicly expands to radiobutton set within radiobutton set:

(here checkboxes but single option required ofc)
- [ ] fork point
- [ ] parent
- [ ] manually picked commit
  - [ ] abc2137 Do a lot of things
  - [ ] 2a1b3c7 Do a lot of things - 1st round of fixes
  - [ ] 020897 Do a lot of things - final commit

Reply:
> list (seems to be anti-patternish to me, but anyway

Could you elaborate? 🤔

Reply:
Where have you seen a list used like that? can you bring some examples of your idea applied? from intellij? of from some website (https://www.intercity.pl/pl/ does not count 😆 )?

Reply:
😅 okay maybe actually a scrollable list with radio boxes isn't the best idea, let's stick to combobox indeed.

I'd suggest another thing however (can be done within a separate issue): to display, within the Override Fork Point dialog, **the list of commits that will be considered unique to the branch, assuming the select fork point**... WDYT?

Reply:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/109077949/194704108-0a496405-8238-436e-a362-1476e4d4d1ab.png">
The latest look. WDYT?

Reply:
Looks okay to me... probably the **subscript** under _Manually picked commit_ is redundant tho